docs/plugins/discovery-gce.asciidoc

```diff
@@ -417,8 +440,8 @@ gcloud compute instances create myesnode2 --image=elasticsearch-1-0-0-RC1 \
 [source,sh]
 --------------------------------------------------
 gcloud compute instances add-metadata myesnode1 \
-       --zone europe-west1-a \
-       --metadata es_port=9301
+  --zone europe-west1-a \
```

Comment:
Should this also use --zone=, --machine-type= and --metadata= (rather than space)?